src/docs/config/swagger.config.ts

```
@@ -0,0 +1,41 @@
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { INestApplication } from '@nestjs/common';
+import { API_VERSIONS, DEFAULT_VERSION } from './api-version.config';
+
+export const setupSwagger = (app: INestApplication) => {
+    const config = new DocumentBuilder()
+        .setTitle('Farmers Marketplace API')
+        .setDescription('API documentation for the Farmers Marketplace platform')
+        .setVersion('1.0')
```

ESLint

[error] Delete `····`

(prettier/prettier)